docs/content/en/docs/reference/packagespec/metallb/v0.12.1.md

```diff
@@ -11,33 +11,69 @@ description: >
-| password | (optional) Password for TCPMD5 authenticated BGP sessions offered by some peers. ||
-| ebgp-multihop | (optional) Whether eBGP multihop is permitted. Note that it is always on in the native BGP mode. ||
-| node-selectors | (optional) The nodes that should connect to this peer. A node matches if at least one of the node selectors matches. Within one selector, a node matches if all the matchers are satisfied. The semantics of each selector are the same as the label- and set-based selectors in Kubernetes, documented at [Labels and Selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/). By default, all nodes are selected.<br/><pre>node-selectors:<br/># Match by label=value<br/>  - match-labels:<br/>    kubernetes.io/hostname: prod-01<br/># Match by 'key OP values' expressions<br/>  - match-expressions:<br/>    key: beta.kubernetes.io/arch<br/>    operator: In<br/>    values: [amd64, arm]</pre>||
-| **bgp-advertisement** |||
-|aggregation-length| (optional) How much you want to aggregate up the IP address before advertising. For example, advertising 1.2.3.4 with aggregation-length=24 would end up advertising 1.2.3.0/24. For the majority of setups, you'll want to keep this at the default of 32, which advertises the entire IP address unmodified. | 32 |
-|aggregation-length-v6| (optional) How much you want to aggregate up the IPv6 address before advertising. For example, advertising 2001:0db8:85a3:0000:0000:8a2e:0370:7334 with aggregation-length-v6=64 would end up advertising 2001:0db8:85a3:0000:0000:0000:0000:0000/64. For the majority of setups, you'll want to keep this at the default of 128, which advertises the entire IP address unmodified. |128|
-|localpref| (optional) The value of the BGP "local preference" attribute for this advertisement. Only used with IBGP peers (i.e. peers where peer-asn is the same as my-asn). ||
-|communities[]| (optional) BGP communities to attach to this advertisement. Communities are given in the standard two-part form **asn:community number**. You can also use alias names. ||
+| IPAddressPools[] | A list of **IPAddressPool**. | None |
+| L2Advertisements[] | A list of **L2Advertisement**. | None |
+| BGPAdvertisements[] | A list of **BGPAdvertisement**. | None |
+| BGPPeers[] | A list of **BGPPeer**. | None |
+|---|---|---|
+|**IPAddressPool**|A list of IP address ranges over which MetalLB has authority. You can list multiple ranges in a single pool, they will all share the same settings. Each range can be either a CIDR prefix, or an explicit start-end range of IPs.||
+| name | Name for the address pool. | None |
+| addresses[] | A list of **string** representing CIRD or IP ranges. | None |
+| autoAssign | AutoAssign flag used to prevent MetallB from automatic allocation for a pool. | true |
+|---|---|---|
+|**L2Advertisement**|L2Advertisement allows to advertise the LoadBalancer IPs provided by the selected pools via L2.||
```

Comment:
Could you add what is being allowed to advertise? For example, "allows MetalLB" or "allows the application"